app/models/order_detail.rb

```diff
@@ -38,7 +38,7 @@ def set_problem_order
   belongs_to :price_policy
   belongs_to :statement, inverse_of: :order_details
   belongs_to :journal
-  belongs_to :order
+  belongs_to :order, inverse_of: :order_details
```

Comment:
We put the reverse in the `Order` model yesterday, but that caused the `Accessories::ChildUpdater` specs to fail. I'd still like that to happen. but I've been looking at it too long and need to step away.